acceptancetests/utility.py

```diff
@@ -317,6 +317,10 @@ def add_basic_testing_arguments(parser, using_jes=False, deadline=True,
     parser.add_argument('--keep-env', action='store_true',
                         help='Keep the Juju environment after the test'
                         ' completes.')
+    parser.add_argument('--existing', action='store_true',
```

Comment:
You should add a check to enforce (or at least mention) the need for --model to go with --existing. That said, do you really need 2 arguments for this? Could --existing just optionally take a target model, with a default?